README.md

```diff
@@ -1,7 +1,7 @@
-# The Rust Programming Language
+# The Rust Programming Language 
```

Comment:
this change seems extraneous.

Reply:
Sorry this was a misclick on my part.
But now that I think of it, a Rust logo wouldn't be bad. Would you like me to create a logo for it?
or like use some badge such as [this](https://img.shields.io/badge/--black?style=for-the-badge&logo=rust) for instance?

Reply:
badge seems nice

Reply:
:D Will add that!

Reply:
I've replaced the h1 heading "The Rust Programming Language" with the badge which says "Rust" beside the Rust logo, with a black background wording of "Programming Language", which you can adjust it to any wording you want by replacing the word before the first dash and after it like `badge/word1-word2-colour?` in the link `https://img.shields.io/badge/Rust-Programming%20Language-black?style=flat&logo=rust` and thereafter select the color you want. (hexcodes work too I think)

I have included an image link instead of the badge notation so you can tweak with parameters such as height width etc upto your preference.

Reply:
![image](https://user-images.githubusercontent.com/155238/85696928-5af4ae80-b6a7-11ea-81c6-a56e2cc0665d.png)